README.md

```diff
@@ -1,38 +1,16 @@
-# Beginners Python Project - Random Name Generator
-Building a Random Name Generator application.
-## Introduction 
-In this project, you will be building a text-based user interface that will take a input and output a list of random names from a pre-defined list. 
-
-## Learning Objects
-- User inputs
-- Validating inputs
-- Importing packages
-- Documenting code 
-- Writing functions
-- Control flows
-- Printing outputs to the user
-- Using random functionality in Python
-- Basic structure of Python Scripts
-
-## Project Brief
-This application with simulate randomly generating names from a pre-defined list. A use case could be: a set of new parents can't choose a name from a shortlist. Why not use Python to help! 
+# Random Name Generator
 
-The application will: 
-- Take a user input of shortlist names.
-- Take a second user input of how many names they want randomly selected from that shortlist. This should be a positive integer. The app should check if the input is valid.
-- Print to the user the randomly selected names. 
-
-## Suggestions
-Feel free to structure the application however you see fit, but here is a project structure recommendation. If you have experience with pseudocode or flowcharts, I would recommend mapping out the project using these tools before you start any programming. 
+## Introduction 
+A simple random name generator application which you can generate within a few types and clicks.
 
-I would also recommend using Visual Studio Code as your editor, but this is personal preference. Make sure you implement coding standards and Pythonic style in the application!
+## Instructions
+1) Enter a name of your desire and click enter.
+2) Click 'x' to pick the number of names you want to select.
```

Comment:
`Click 'x' to pick the number of names you want to select.` This sounds like you are talking about closing the window.